bedrock/products/templates/products/vpn/more/ip-address.html

```diff
@@ -40,5 +40,5 @@ <h3>{{ ftl('vpn-ip-address-should-you-hide') }}</h3>
 <p>{{ ftl('vpn-ip-address-there-are-also') }}</p>
 
 <h3>{{ ftl('vpn-ip-address-how-do-you') }}</h3>
-<p>{{ ftl('vpn-ip-address-a-vpn-is', vpn=url('products.vpn.more.when-to-use'), mozvpn=url('products.vpn.landing')) }}</p>
+<p>{{ ftl('vpn-ip-address-a-vpn-is', vpn=url('products.vpn.more.when-to-use'), mozvpn=url('products.vpn.landing'), countries='30+') }}</p>
```

Comment:
Suggest using `settings.VPN_CONNECT_COUNTRIES` for the variable here. I would also leave the `+`as part of the localized string, as translators may want to use another way to describe things.